modules/c++/math.linear/include/math/linear/MatrixMxN.h

```diff
@@ -39,7 +39,17 @@ template<typename _T> bool equals(const _T& e1, const _T& e2)
 
 template<typename _T> inline bool equals(const _T& e1, const _T& e2, _T eps)
 {
+#ifdef _MSC_VER // Visual Studio
```

Comment:
Similarly, to avoid needing to sprinkle this through all our code, can we just put in `/wd4127`?